src/app/public/modules/text-editor/menubar/text-editor-menubar.component.ts

```diff
@@ -0,0 +1,190 @@
+import {
+  Component,
+  Input,
+  ViewEncapsulation,
+  OnInit,
+  OnDestroy,
+  ChangeDetectionStrategy
+} from '@angular/core';
+
+import {
+  Subject
+} from 'rxjs';
+
+import {
+  takeUntil
+} from 'rxjs/operators';
+
+import {
+  SkyTextEditorMenubarAction
+} from '../types/menubar-action';
+
+import {
+  SkyTextEditorService
+} from '../services/text-editor.service';
+
+import {
+  SkyTextEditorMergeField
+} from '../types/text-editor-merge-field';
+
+import {
+  SkyTextMergeFieldService
+} from '../services/text-merge-field.service';
+
+import {
+  SkyDropdownMessage,
+  SkyDropdownMessageType
+} from '@skyux/popovers';
+
+/**
+ * @internal
+ */
+@Component({
+  selector: 'sky-text-editor-menubar',
+  templateUrl: './text-editor-menubar.component.html',
+  styleUrls: ['./text-editor-menubar.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None
+})
+export class SkyTextEditorMenubarComponent implements OnDestroy, OnInit {
+
+  @Input()
```

Comment:
Do the public inputs in this file need JSDocs comments?

Reply:
Nope. This is an internal component.